edx_django_utils/tests/test_private_utils.py

```diff
@@ -19,6 +21,7 @@ def test_check_middleware_dependencies_simple_success(self):
         middleware = _TestMiddleware()
         _check_middleware_dependencies(middleware, required_middleware=['required.Middleware'])
 
+    @unittest.skipIf(django.VERSION[0] == 2, 'MIDDLEWARE_CLASSES only works with less than django-2 versions.')
```

Comment:
This test requires only for less than django2 versions.

Reply:
Django 3.0 is already in alpha testing; `django.VERSION[0] > 1` is probably a better condition to use here.